content/en/_index.md

```diff
@@ -245,7 +242,11 @@ use Google\Cloud\Dlp\V2\DlpServiceClient;
  * @param string $filter        (Optional) filter to use
  * @param string $languageCode  (Optional) language code, empty for 'en-US'
  */
-function list_info_types(string $filter = '', string $languageCode = ''): void
+function list_info_types(
+    // TODO(developer): Replace these values before running your sample: 
+    string $filter = 'your-filter', 
+    string $languageCode = 'your-language-code'): void
```

Comment:
I don't like having defaults that will break the code.... This is not how our samples work today and it's not how we want them to look going forward. What we had before was better for two reasons:

1. These functions are executable on the CLI, and if the user does not provide a value for them (in this case), they will throw errors
2. The above example really doesn't add anything helpful... It doesn't show an example of a valid filter OR valid language code. So it's completely pointless (IMHO). If we want to provide good examples of what these. values could be, we can do so in the PHPDoc of the parameter (as we do above for language code).

What we have here is not idiomatic to PHP, nor is it helpful